.travis.yml

```diff
@@ -3,6 +3,7 @@ sudo: false
 python:
     - 2.7
     - pypy
+    - 3.6
```

Comment:
Any reason not to add 3.4/3.5 as well?

Reply:
We could do 3.5 but '%' operator for bytes is probably a problem on 3.4.

Reply:
That is a good reason.